.github/pull_request_template.md

```diff
@@ -1,16 +1,29 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-If this PR closes another issue, add 'closes #<issue number>' somewhere in the PR summary. GitHub will automatically close that issue when this PR gets merged. Alternatively, adding 'refs #<issue number>' will not close the issue, but help provide the reviewer more context.-->
+
+## What this PR does
+
+_Delete this text and tell us why we need this change._
+
+<!--  
+Thanks for sending a pull request!  
+Here are some tips for you:
+If this PR closes another issue, add 'closes #<issue number>' somewhere in the PR summary. GitHub will automatically close that issue when this PR gets merged. 
+Alternatively, adding 'refs #<issue number>' will not close the issue, but help provide the reviewer more context.
 
 Closes #[issue number]
+-->
 
-**What this PR does / why we need it**:
+### Special notes
```

Comment:
I'm not sure I've seen people use this much... should we just remove this section?

Reply:
Alternatively we could put it inside `<!--` section and then leave it commented out and have people opt in if they do have special notes?

Reply:
Did you make this change?

I meant put the whole `### Special notes` section in comments (so it doesn't show up unless somebody uncomments it).

Reply:
Now updated.